discord/partial_emoji.py

```diff
@@ -104,13 +105,13 @@ def from_dict(cls: Type[PE], data: Dict[str, Any]) -> PE:
             name=data.get('name', ''),
         )
 
-    def to_dict(self) -> Dict[str, Any]:
+    def to_dict(self) -> PartialEmojiPayload:
```

Comment:
This is wrong. There's a reason I did it this way because the types aren't the same.